packages/turbo-test-utils/src/useFixtures.ts

```diff
@@ -12,37 +19,45 @@ interface SetupTextFixtures {
   };
 }
 
-export default function setupTestFixtures({
+export function setupTestFixtures({
   directory,
   test = "",
   options = {},
 }: SetupTextFixtures) {
   const fixtures: Array<string> = [];
-  const parentDirectory = path.join(directory, test ? test : "test-runs");
+  const parentDirectory = path.join(directory, test ? test : uuidv4());
 
-  afterEach(() => {
-    fixtures.forEach((fixture) => {
-      fs.rmSync(fixture, { recursive: true, force: true });
-    });
+  afterEach(async () => {
+    await Promise.all(
+      fixtures.map((fixture) =>
+        rimraf(fixture, {
```

Comment:
When running a lot of tests (workspaces runs like 500 in parallel) this can sometimes hit a busy signal. This makes that a bit more resilient.